test.js

```diff
@@ -1,35 +1,63 @@
+const versions = fs.readdirSync(contentDir)
+const originalSourceLocale = 'en-US'
 
-test('includes source English content for all major versions', () => {
-  const fetchedMajors = fs.readdirSync(contentDir).sort()
-  const targetMajors = supportedVersions.sort()
-  expect(targetMajors).toEqual(fetchedMajors)
+describe('original content', () => {
+  test('exist for all supported versions', () => {
+    expect(supportedVersions.sort()).toEqual(versions.sort())
+    versions.forEach((major) => {
+      const languages = fs.readdirSync(path.join(contentDir, major))
+      expect(languages.includes(originalSourceLocale)).toBe(true)
+    })
+  })
 
-  fetchedMajors.forEach(major => {
-    const languages = fs.readdirSync(path.join(contentDir, major))
-    expect(languages.includes('en-US')).toBe(true)
+  test('includes only markdown files', () => {
+    expect(versions.length).toBeGreaterThan(1)
+    versions.forEach((version) => {
+      const docsDir = path.join(
+        contentDir,
+        version,
+        originalSourceLocale,
+        'doc'
+      )
+      const files = walk(docsDir, { directories: false })
+      expect(files.length).toBeGreaterThan(0)
+      expect(files.every((file) => file.relativePath.endsWith('.md')))
+    })
   })
 })
 
-xtest('includes translated content for all major versions', () => {
-  // pending Crowdin integration
-})
+describe('translated content', () => {
+  test('saves original structure', () => {
+    versions.forEach((version) => {
+      const languages = fs.readdirSync(path.join(contentDir, version))
+      const originalPath = path.join(
+        contentDir,
+        version,
+        originalSourceLocale,
+        'doc'
+      )
+      const originalFiles = walk(originalPath, { directories: false }).map(
+        ({ relativePath }) => relativePath
+      )
+      languages.forEach((language) => {
+        const translatedPath = path.join(contentDir, version, language, 'doc')
+        const translatedFiles = walk(translatedPath, {
+          directories: false
+        }).map(({ relativePath }) => relativePath)
+        const translatedOriginDiff = difference(translatedFiles, originalFiles)
 
-test('includes only markdown files, ignoring images and other files', () => {
-  const versions = fs.readdirSync(contentDir)
-  expect(versions.length).toBeGreaterThan(1)
-  versions.forEach(version => {
-    const languages = fs.readdirSync(path.join(contentDir, version))
-    expect(languages.length).toBeGreaterThan(0)
-    languages.forEach(language => {
-      const docsDir = path.join(contentDir, version, language, 'doc')
-      const files = walk(docsDir, { directories: false })
-      expect(files.length).toBeGreaterThan(0)
-      expect(files.every(file => file.relativePath.endsWith('.md')))
+        // can be used to remove mismatch files
+        // translatedOriginDiff.map(filePath => fs.remove(path.join(translatedPath, filePath)))
+
+        expect(translatedOriginDiff.length).toBe(0)
```

Comment:
```suggestion
        expect(translatedOriginDiff).toEqual([])
```

Reply:
If this fails the output will not be very useful, as it will not show the diff. My suggestion should help by showing the diffed array content in the failure output.